packages/viewer/src/lib/types/frame.ts

```diff
@@ -29,8 +30,15 @@ export class Frame {
     public readonly image: FrameImage,
     public readonly scene: FrameScene,
     private readonly depthBufferBytes: Uint8Array | undefined,
-    private readonly featureMapBytes: Uint8Array | undefined
-  ) {}
+    private readonly featureMapBytes: Uint8Array | undefined,
+    private readonly id = UUID.create()
+  ) {
+    this.id = UUID.create();
```

Comment:
Looks like this will clear out any value passed to the constructor, can we just rely on the defaulted value?